commit/merkleroot/outcome.go

```diff
@@ -223,10 +222,21 @@ func filterRootsBasedOnRmnSigs(
 	signedLaneUpdates []*rmnpb.FixedDestLaneUpdate,
 	roots []cciptypes.MerkleRootChain,
 	rmnEnabledChains map[cciptypes.ChainSelector]bool,
+	addressCodec cciptypes.AddressCodec,
 ) []cciptypes.MerkleRootChain {
 	// Create a set of signed roots for quick lookup.
 	signedRoots := mapset.NewSet[rootKey]()
 	for _, laneUpdate := range signedLaneUpdates {
+		addrStr, err := addressCodec.AddressBytesToString(
+			laneUpdate.LaneSource.OnrampAddress,
+			cciptypes.ChainSelector(laneUpdate.LaneSource.SourceChainSelector))
+		if err != nil {
+			lggr.Errorw("can't convert Onramp address to string",
```

Comment:
Change to critical error on all failures of the addressCodec

Reply:
The chainlink-common logger doesn't have critical